package.json

```diff
@@ -127,9 +127,9 @@
     "regenerator-runtime": "^0.13.9",
     "require-reload": "^0.2.2",
     "simple-git": "^2.47.1",
-    "styled-components": "^5.3.1",
+    "styled-components": "5.3.1",
     "tarball-extract": "^0.0.6",
-    "ts-jest": "^27.0.5",
```

Comment:
took out the carrot because when I ran `yarn` it was trying to update these I have them in a separate PR